ember_debug/promise-debug.js

```diff
@@ -1,8 +1,8 @@
 import PortMixin from 'ember-debug/mixins/port-mixin';
 import PromiseAssembler from 'ember-debug/libs/promise-assembler';
+import computed from 'ember-new-computed';
```

Comment:
Unfortunately we can't use addons in `ember_debug` since it's not part of the Ember app (this is the part that's injected into the app being debugged). The `ember-new-computed` module does not exist here. The tests are accidentally passing due to module leaks between the Ember app and ember_debug.